File: app/src/main/java/com/jetsynthesys/rightlife/ui/payment/Service.java
package com.jetsynthesys.rightlife.ui.payment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Service {

   @SerializedName("_id")
   @Expose
   private String id;
   @SerializedName("moduleId")
   @Expose
   private String moduleId;
   @SerializedName("title")
   @Expose
   private String title;
   @SerializedName("moduleImageUrl")
   @Expose
   private String moduleImageUrl;

   public String getId() {
      return id;
   }

   public void setId(String id) {
      this.id = id;
   }

   public String getModuleId() {
      return moduleId;
   }

   public void setModuleId(String moduleId) {
      this.moduleId = moduleId;
   }

   public String getTitle() {
      return title;
   }

   public void setTitle(String title) {
      this.title = title;
   }

   public String getModuleImageUrl() {
      return moduleImageUrl;
   }

   public void setModuleImageUrl(String moduleImageUrl) {
      this.moduleImageUrl = moduleImageUrl;
   }

}

File: app/src/main/java/com/jetsynthesys/rightlife/ui/healthcam/Service.kt
package com.jetsynthesys.rightlife.ui.healthcam;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Service {

    @SerializedName("identifier")
    @Expose
    private String identifier;
    @SerializedName("measurementId")
    @Expose
    private String measurementId;
    @SerializedName("scanUrl")
    @Expose
    private String scanUrl;

    public String getIdentifier() {
        return identifier;
    }

    public void setIdentifier(String identifier) {
        this.identifier = identifier;
    }

    public String getMeasurementId() {
        return measurementId;
    }

    public void setMeasurementId(String measurementId) {
        this.measurementId = measurementId;
    }

    public String getScanUrl() {
        return scanUrl;
    }

    public void setScanUrl(String scanUrl) {
        this.scanUrl = scanUrl;
    }

}